### ISSUE TITLE
Other Work this Sprint (Sprint #17)

### ISSUE THREAD

**Issue by schancase** [cds-snc/notification-planning#2941]
This is our "Other Work Doing or Done" Bucket will be created on a per-Sprint or multi-sprint Basis in order to:

* Bring visibility at the feature level to other work we are doing;
* Prompt us to examine the list of child items (i.e. Tasks) to see if in fact they should be parented elsewhere OR whether a bunch of them on their own merit their own feature; and
* Improve workflow metrics for showing work the team got done each sprint at the feature level

**Comment by schancase** [comment]
moving this feature to 'for team refinement' so we can sync as a team on which of these items we'd like to keep here vs move out into a future bucket. cc @connorscarolyns

**Comment by schancase** [comment]
update to previous comment. I've not created a sprint #18 bucket (#2985) and moved some there that did not arise in our sprint planning sessions this week. Would be still good to validate this proposed bucket with the team

**Comment by schancase** [comment]
closing as done
